apps/desktop/src/components/v3/IntegrateUpstreamModal.svelte

```diff
@@ -64,15 +64,16 @@
 	let branchStatuses = $state<StackStatusesWithBranchesV3 | undefined>();
 	const stackService = getContext(StackService);
 	let appliedBranches = $state<string[]>();
-	let filteredReviews = $state<PullRequest[] | undefined>(undefined);
+	// Any PRs belonging to applied branches that have been merged
+	let filteredReviews = $state<PullRequest[]>([]);
 	const reviewMap = $derived(new Map(filteredReviews?.map((r) => [r.sourceBranch, r])));
 
 	const isDivergedResolved = $derived(base?.diverged && !baseResolutionApproach);
 	const [integrateUpstream] = $derived(upstreamIntegrationService.integrateUpstream(projectId));
 
 	$effect(() => {
 		if (!modal?.imports.open) return;
-		if (branchStatuses?.type !== 'updatesRequired' || filteredReviews === undefined) {
+		if (branchStatuses?.type !== 'updatesRequired') {
```

Comment:
This was a very flawed condition that was trying to express "If the filtered reviews are loading, don't bother with anything else". This broke down when there were no forge integrations.